packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -315,7 +315,8 @@ class ThemeData with Diagnosticable {
     BottomNavigationBarThemeData? bottomNavigationBarTheme,
     BottomSheetThemeData? bottomSheetTheme,
     ButtonThemeData? buttonTheme,
-    CardTheme? cardTheme,
+    // TODO(QuncCccccc): Change the parameter type to CardThemeData
+    Object? cardTheme,
```

Comment:
Similar to #155129, to help smooth rolling into G3, I updated the parameter type in `ThemeData()` and `ThemeData.copyWith()` to `object?` and added a TODO to change the parameter type back to `CardThemeData`.